R/cluster.R

```diff
@@ -203,13 +204,33 @@ makeCluster <-
       commandLine = commandLine
     )
 
-    pool <- rAzureBatch::getPool(poolConfig$name)
-
     if (grepl("AuthenticationFailed", response)) {
       stop("Check your credentials and try again.")
+    }
+
+    if (grepl("PoolBeingDeleted", response)) {
+      pool <- rAzureBatch::getPool(poolConfig$name)
+
+      cat("Waiting for deleting the previous pool. Should take less than 10 minutes\n")
```

Comment:
"Cluster 'foo' already exists and is being deleted. Another cluster with the same name cannot be created until it is deleted. Please wait for the cluster to be deleted or create one with a different name"

Reply:
So I said cluster here, but I feel that sometimes we use cluster and sometime we use pool. We need to make sure all messages and standard, and match the config.